spec/services/publishers/vacancy_sorting_options_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe Publishers::VacancySortingOptions do
+  subject { described_class.new(organisation, vacancy_type) }
+
+  context "for a School" do
```

Comment:
I like the safety net the consistency of "contexts always use `when`" provides, but if we say it's ok to just use sensible descriptions then I am ok with that

Reply:
Good point - will refactor this momentarily when I do the same for the public-facing sort options!